packages/taquito/src/subscribe/observable-subscription.ts

```diff
@@ -23,7 +24,7 @@ export class ObservableSubscription<T> implements Subscription<T> {
             this.call(this.closeListeners);
           }
         ),
-        this.shouldRetry ? retry() : tap(),
+        this.shouldRetry ? retryWhen(errors => errors.pipe(delay(1000), take(10))) : tap(),
```

Comment:
Should the retry strategy configurable by the user instead of hard-coding 10 retries with a 1-second delay?

Reply:
I have made these user configurable.